pages/seed_db.php

```diff
@@ -45,7 +45,7 @@ function map_gender(string $g): Gender
     $newUser->licence = $user['num_lic'];
     $newUser->nose_email = $user['email'];
     $newUser->real_email = $user['realmail'];
-    $newUser->phone = $user['telport'];
+    $newUser->phone = preg_replace("/[^0-9]/", "", $user['telport']);
```

Comment:
phones can have a + in the number. Maybe we need to change the model instead 🤔

Reply:
Yes that's true, especially if we have a foreign user